tests/dm/toml.dme

```diff
@@ -17,5 +17,5 @@ var/test_json = @{"
 
     var/toml_output = rustg_read_toml_file("test.toml")
 
-    if (toml_output != test_json)
+    if (json_decode(toml_output) ~= json_decode(test_json)) // We want to compare the decoded *objects*
```

Comment:
json_decode(object) ~= json_decode(string)

You either want to compare toml_output (object) to json_decode(test_json) (object) or re-encode both of those into json text and compare the strings (Which is how it worked prior to #98)

Reply:
yeah it was just midnight and i was extremely tired, like my eyes were involuntarily closing